### PR TITLE
Updated install-kubectl-windows.md

### PR DESCRIPTION
**Pull request by i-am-yuvi**
I have addded a better view of the kubectl version output by adding "--output=yaml" so that the new ones will be able to understand the output easily.

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -66,6 +66,11 @@ The following methods exist for installing kubectl on Windows:
    ```cmd
    kubectl version --client
    ```
+   Or use this for detailed view of version:
+
+   ```cmd
+   kubectl version --client --output=yaml
+   ```
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:73]: In my view: if we make this change, we should make the equivalent change to the Windows and Mac OS docs too.

Comment (i-am-yuvi) [reply content/en/docs/tasks/tools/install-kubectl-windows.md:73]: It should be linux and mac OSinstead. Since I have changed in Windows.

Comment (sftim) [reply content/en/docs/tasks/tools/install-kubectl-windows.md:73]: Ah, yep.

Comment (jihoon-seo) [reply content/en/docs/tasks/tools/install-kubectl-windows.md:73]: Pages:
- https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
- https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/
- https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/

**Comment by sftim** [comment]
@tengqm, should we make the equivalent change to Linux and macOS documentation?

**Comment by tengqm** [comment]
> @tengqm, should we make the equivalent change to Linux and macOS documentation?

For consistency, I'd like to see all instances fixed in a single PR.

**Pull request closed without merging**